### PR TITLE
docs(ifr): add data processing information

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Comment by bene2k1** [comment]
do not merge before validated by legal team

**Comment by tgenaitay** [comment]
Message de la légal team

> Nous n'avons pas de compte Github au service legal pour valider la policy, je valide donc via ce ticket la politique privacy destinée à la documentation.

**Review by seiteta — CHANGES_REQUESTED** [review]

File: ai-data/managed-inference/reference-content/data-privacy-security-scaleway-ai-services.mdx
Comment (bene2k1) [reply ai-data/managed-inference/reference-content/data-privacy-security-scaleway-ai-services.mdx:21]: @seiteta do you mean the whole part "Data processing" or just the information about Input Data / Metadata ?

Comment (seiteta) [reply ai-data/managed-inference/reference-content/data-privacy-security-scaleway-ai-services.mdx]: @bene2k1: please find my suggestion in my [commit ](https://github.com/scaleway/docs-content/pull/3190/commits/10c85daaa81acb82653b40985d2bdd30ca741bf1)

**Comment by bene2k1** [comment]
@tgenaitay we can merge this if good for you.

**Review by ldecarvalho-doc — APPROVED** [review]

File: ai-data/managed-inference/reference-content/data-privacy-security-scaleway-ai-services.mdx
Comment (nerda-codes) [inline ai-data/managed-inference/reference-content/data-privacy-security-scaleway-ai-services.mdx:15]:
```suggestion
Scaleway's AI services implement robust security measures to ensure customer data privacy and integrity. Below is an overview of these practices.
```

**Review by nerda-codes — APPROVED** [review]

**Comment by tgenaitay** [comment]
Well done! 

Was wondering if it’s worth putting some words about our compliance with RGPD and how our AI services not obliged by extra territorial laws such as the American Cloud Act.
My feeling is that our EU customers see it as advantage for Scaleway overall, but whoever reads this page in the future (data teams new to Scaleway) could use a clear statement in the context of AI. 

https://fr.linkedin.com/pulse/app-et-infrastructure-quelle-souverainet%C3%A9-mieux-pour-choisir-morales-0upde

**Review by seiteta — APPROVED** [review]

**Comment by tgenaitay** [comment]
Perfect to start with! Many thanks.

**Review by tgenaitay — APPROVED** [review]

**Pull request merged**
